lib/location/method_gnss.c

```diff
@@ -264,12 +264,7 @@ static void method_gnss_agps_request_work_fn(struct k_work *item)
 						     NRF_CLOUD_REST_AGPS_REQ_CUSTOM,
 						     &agps_request,
 						     NULL,
-#if defined(CONFIG_NRF_CLOUD_AGPS_FILTERED_RUNTIME)
-						     true,
-						     CONFIG_NRF_CLOUD_AGPS_ELEVATION_MASK
-#else
 						     false, 0
```

Comment:
To align with the style of above lines, this should be split although it wasn't in the past:
```suggestion
						     false,
						     0
```

Reply:
True.